### PR TITLE
Feacture/sem-018

### PR DESCRIPTION
**Pull request by AlexTheLion99**
.

### CODE REVIEW

File: src/ui/components/atoms/checkbox/main.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+          options: {
+            presets: [require.resolve('babel-preset-react-app')],
+          },
+        },
+      ],
+    });
+    config.resolve.extensions.push('.ts', '.tsx');
+    return config;
+  },
+};
```
Comment (Ditmar) [inline src/ui/components/atoms/checkbox/main.js:19]: Why do you need this config, this config is made in the storybook please remove it!

**Review by Ditmar — CHANGES_REQUESTED** [review]
read the comments

File: src/ui/components/atoms/checkbox/main.js (hunk 1 above)
Comment (AlexTheLion99) [reply src/ui/components/atoms/checkbox/main.js:19]: The setting was to ensure that TypeScript and TSX files are recognized correctly. But I realized that it is already integrated into the storybook, so it is no longer necessary.

**Comment by AlexTheLion99** [comment]
Solicito revisión de mi PR:
https://ditmar.github.io/backoffice-openscience/branch-feacture/SEM-018/?path=/story/ui-components-demo-atoms-checkbox--focus-selected

**Pull request closed without merging**